README.md

```diff
@@ -18,51 +18,94 @@ Using `any-agent`, you can more easily switch to a new or different agent framew
 
 any-agent also provides a 'trace-first' [llm-as-a-judge powered evaluation tool](https://mozilla-ai.github.io/any-agent/evaluation/) for flexible evaluation of agent execution traces.
 
-## Supported Frameworks
+## [Supported Frameworks](https://mozilla-ai.github.io/any-agent/frameworks/)
 
 [Google ADK](https://github.com/google/adk-python) [![LangChain](https://img.shields.io/badge/LangChain-1e4545?logo=langchain&logoColor=white)](https://github.com/langchain-ai/langgraph) [![LlamaIndex](https://img.shields.io/badge/🦙%20LlamaIndex-fbcfe2)](https://github.com/run-llama/llama_index) [![OpenAI Agents](https://img.shields.io/badge/OpenAI%20Agents-black?logo=openai)](https://github.com/openai/openai-agents-python) [![Smolagents](https://img.shields.io/badge/Smolagents-ffcb3a?logo=huggingface&logoColor=white)](https://smolagents.org/) [Agno AI](https://docs.agno.com/introduction)
 
-## Planned for Support (Contributions Welcome!)
+### Planned for Support (Contributions Welcome!)
 [AWS Bedrock Agents](https://github.com/mozilla-ai/any-agent/issues/16),
 [Pydantic AI](https://github.com/mozilla-ai/any-agent/issues/31),
 [Microsoft AutoGen](https://github.com/mozilla-ai/any-agent/issues/30),
 [Crew AI](https://github.com/mozilla-ai/any-agent/issues/17)
 
-
 ## Quickstart
 
-Refer to pyproject.toml for a list of the extras available. Update your pip install command to include all of the frameworks that you plan on using.
+Refer to [pyproject.toml](./pyproject.toml) for a list of the extras available. Update your pip install command to include
+the frameworks that you plan on using.
+
 ```bash
-pip install 'any-agent[smolagents,langchain,llama_index,openai,mcp]'
+pip install 'any-agent[agno,google,openai]'
```

Comment:
Why did you change this?

Reply:
I am not sure if just put `all`, an arbitrary list of a couple of frameworks or just one (I think is easy to get the point). 
Just feels annoying to update 

wdyt about `all` or the other options?

Reply:
Yeah good point. How about make it something like

```
pip install 'any-agent[all]' # refer to pyproject.toml for individual agent installation options
```